File: Pratica 3 - Estrutura de Repeticao/java_logic/Lista_nome.java
import java.util.ArrayList;

public class Lista_nome {
    public static void main(String[] args) {
        ArrayList<String> nomes = new ArrayList<>();
        nomes.add("Ana");
        nomes.add("Bruno");
        nomes.add("Carla");

        System.out.println("Interando sobre uma lista em Java:");
        for (String nome : nomes) {
            System.out.println(nome);
        }
    }
}

File: Pratica 3 - Estrutura de Repeticao/cpp_logic/lista_nome.cpp
#include <iostream>
#include <vector>
#include <string>
using namespace std;

int main() {
    vector<string> nomes = {"Ana", "Bruno", "Carla"};
    cout << "Interando sobre um vector em C++:" << endl;
    for (const string& nome : nomes) {
        cout << nome << endl;
    }
    return 0;
}